#goal definition
char[] inputText
char[] timeStamp
---
#result definition
bool isOK
---
#feedback
int32[] sequence
